README.md

```diff
@@ -70,12 +74,27 @@ if __name__ == '__main__':
 ```
 
 Once these changes are in place, you can submit the script to AzureML by supplying an additional `--azureml` flag
-on the commandline, like `python myscript.py --azureml`.
-
-That's it!
-
-For details, please refer to the [onboarding page](docs/source/first_steps.md).
+on the commandline, like `python myscript.py --azureml`
  
+In the added code, you need to supply the following parameters:
+* `entry_script`: The script that should be run
+* `snapshot_root_directory`: The directory that contains all code that should be packaged and sent to AzureML. All
+Python code that the script uses must be copied over.
+* `compute_cluster_name`: The name of the AzureML cluster that should run the job. This can be a cluster with CPU or 
+GPU machines. See [here for documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-studio#amlcompute)
```

Comment:
I had removed that to make the readme more concise, I'd rather have the messy bits in the docu. Did you work off the latest main?